Gruntfile.js

```diff
@@ -156,7 +156,9 @@ module.exports = function(grunt) {
                     'frontend/express/public/core/session-frequency/javascripts/countly.models.js',
                     'frontend/express/public/core/session-frequency/javascripts/countly.views.js',
                     'frontend/express/public/core/events/javascripts/countly.models.js',
-                    'frontend/express/public/core/events/javascripts/countly.views.js'
+                    'frontend/express/public/core/events/javascripts/countly.views.js',
+                    'frontend/express/public/core/events/javascripts/countly.events.models.js',
+                    'frontend/express/public/core/events/javascripts/countly.events.views.js'
                 ],
```

Comment:
what is the difference between 
`/core/events/javascripts/countly.views.js`
and 
`/core/events/javascripts/countly.events.views.js`

Reply:
countly.views.js - It is for overview page
countly.events.views.js - It is for detail page

Reply:
hmmm i see, can we rename them so that, the names are more suitable for what they actually represent.
since they are already in the events directory,
maybe we can rename them to `countly.overview.views.js` and `countly.detail.views.js`

Reply:
Sure, will do:)